nixos/modules/services/security/vault.nix

```diff
@@ -63,76 +53,149 @@ in
         '';
       };
 
-      address = mkOption {
-        type = types.str;
-        default = "127.0.0.1:8200";
-        description = "The name of the ip interface to listen to";
-      };
-
-      tlsCertFile = mkOption {
-        type = types.nullOr types.str;
-        default = null;
-        example = "/path/to/your/cert.pem";
-        description = "TLS certificate file. TLS will be disabled unless this option is set";
-      };
-
-      tlsKeyFile = mkOption {
-        type = types.nullOr types.str;
-        default = null;
-        example = "/path/to/your/key.pem";
-        description = "TLS private key file. TLS will be disabled unless this option is set";
-      };
-
-      listenerExtraConfig = mkOption {
-        type = types.lines;
-        default = ''
-          tls_min_version = "tls12"
```

Comment:
this was removed because this is already the default value `vault` uses and therefore redundant

source: https://developer.hashicorp.com/vault/docs/configuration/listener/tcp#tls_min_version